nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -1655,49 +1668,6 @@ public void suggestNullableArgumentOnBytecodeClassFileInfoOnly() {
     }
   }
 
-  @Test
-  public void verifySerializationVersionIsSerialized() {
```

Comment:
Why is this test being removed?

Reply:
Should we still serialize the version NullAway is configured ? That seemed redundant as we are setting the version ourself in the configurations. Please let me know if you would like to have it back. Thank you.

Reply:
Ah, the idea is that now, instead of the auto-annotator checking which version NullAway is using for data serialization, it _asks_ for a specific version?

So, here is the problem, what happens with existing versions of the auto-annotator, which need to still figure out that they are dealing with e.g. version 1 of the format vs version 0.

Actually, if the fix serialization version is being loaded from XML configuration, rather than directly from an Error Prone option... how do I downgrade the version to configure this to work with the existing v1.3.4 of the auto-annotator, which is the entire goal of this compatibility?

Reply:
Brought is back as we discussed offline and we can set the serialization version via error prone flags.

a8bac9fc0f856b1cbf82ceb4b22fdf0675f69dc9